roles/shiftstack/tasks/test_config.yml

```diff
@@ -48,12 +49,19 @@
 
   always:
     - name: "Copy the artifacts from the pod '{{ cifmw_shiftstack_client_pod_name }}'"
+      vars:
+        _exclude_options: >-
+          {{
+            '--exclude=' ~ cifmw_shiftstack_exclude_artifacts_regex
+            if cifmw_shiftstack_exclude_artifacts_regex | default('') != ''
```

Comment:
Just realized that if `cifmw_shiftstack_exclude_artifacts_regex` is optional now, we can set it to empty for the molecule test (in [roles/shiftstack/molecule/default/converge.yml](https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/shiftstack/molecule/default/converge.yml)) as there is no openshift-install artifacts to exclude in this test.

Reply:
Good one! Added